internal/consensus/types/round_state.go

```diff
@@ -82,11 +82,11 @@ type RoundState struct {
 	LockedBlockParts    *types.PartSet      `json:"locked_block_parts"`
 
 	// Last known round with POL for non-nil valid block.
-	ValidRound int32        `json:"valid_round"`
-	ValidBlock *types.Block `json:"valid_block"` // Last known block of POL mentioned above.
+	TwoThirdPrevoteRound int32        `json:"valid_round"`
```

Comment:
I don't think we should change these variable names since they are somewhat specific to the tendermint paper.

Reply:
I can explain the reasoning behind this decision. Simply the author of some of the original PR's was confused as to why this field was called `Valid` when it didn't really relate at all to the validity of a block. I'm pretty sure `ValidRound` and `ValidBlock` are when a block receives 2/3 prevote (can't remember what the difference is between `LockedBlock` and `LockedRound`). But the block could still be invalid to what the application thinks - which is now relevant since the introduction of `ProcessProposal`, and it could be invalid with respect to Tendermint. I think you're idea that it should be named as in the paper (or better still as it is in the spec) seems right, but I think we should add more of a description in the comments

Reply:
Thanks @cmwaters for chiming in. As discussed in today's meeting, I will address this thread by following @williambanfield's suggestion of reverting the renaming, but will add an explanation (based on your text above) to the place where those variables are declared